cucumber/kubernetes/features/step_definitions/login_steps.rb

```diff
@@ -35,7 +35,7 @@ def login_with_custom_prefix request_ip, host_id_suffix, host_id_prefix, success
 
 def login_with_username request_ip, username, success, headers = {}
   begin
-    @pkey = OpenSSL::PKey::RSA.new 1048
```

Comment:
it's weird because 1048 isn't a power of 2...

Reply:
lol indeed - this is weird